packages/environment-check/src/checks/get-installed.ts

```diff
@@ -152,3 +152,22 @@ export async function getFioriGenVersion(): Promise<string> {
     }
     return fioriGenVersion;
 }
+
+/**
+ * Returns the versions of node.js modules.
+ *
+ * @returns modules and versions
+ */
+export async function getProcessVersions(): Promise<{ [id: string]: string }> {
+    const output = await spawnCommand('node', ['-p', 'process.versions']);
+    const versions = output.split('\n');
```

Comment:
When I execute following command in terminal
```
$ node -p process.versions
```
I get as result
```
{
  node: '14.20.0',
  v8: '8.4.371.23-node.87',
  uv: '1.42.0',
  zlib: '1.2.11',
  brotli: '1.0.9',
  ares: '1.18.1',
  modules: '83',
  nghttp2: '1.42.0',
  napi: '8',
  llhttp: '2.1.5',
  openssl: '1.1.1q',
  cldr: '40.0',
  icu: '70.1',
  tz: '2021a3',
  unicode: '14.0'
}
```

Would it not be easier to do a `JSON.parse()` of the output instead of applying string operations like `split`, `trim`, `indexOf`?

Reply:
I had hoped it would be easy to parse the output without much manipulation. But the string returned cannot be parsed as keys are not quoted (`SyntaxError: Unexpected token n in JSON at position 3)`. 

The returned string contains new lines - see after using `JSON.stringify()`

`"{\n  node: '16.17.0',\n  v8: '9.4.146.26-node.22',\n  uv: '1.43.0',\n  zlib: '1.2.11',\n  brotli: '1.0.9',\n  ares: '1.18.1',\n  modules: '93',\n  nghttp2: '1.47.0',\n  napi: '8',\n  llhttp: '6.0.7',\n  openssl: '1.1.1q+quic',\n  cldr: '41.0',\n  icu: '71.1',\n  tz: '2022a',\n  unicode: '14.0',\n  ngtcp2: '0.1.0-DEV',\n  nghttp3: '0.1.0-DEV'\n}\n"`

Am I missing a trick here?

Reply:
The string contains '\n' not a new line.  The new line characters have already been escaped into the string.

Reply:
@cianmSAP, what about running the command as `node -p 'JSON.stringify(process.versions)'` and parsing the output?

Reply:
thanks @Klaus-Keller , that works